projects/core/src/routing/event/routing-event.module.ts

```diff
@@ -0,0 +1,7 @@
+import { NgModule } from '@angular/core';
+import { RoutingEventBuilder } from './routing-event.builder';
+
+@NgModule()
```

Comment:
Let's specify an empty object `{}` inside of the `NgModule()`

Reply:
done